File: ClientApp/src/app/settings/cameras/camera.ts
export class Camera {
    latitude: number;
    longitude: number;
    manufacturer: Manufacturer;
    modelNumber: string;
    openAlprName: string;
    openAlprCameraId: number;
    cameraPassword: string;
    cameraUsername: string;
    updateOverlayTextUrl: string;
    platesSeen: number;
    sampleImageUrl: string;

    constructor(init?:Partial<Camera>) {
        Object.assign(this, init);
    }
}

export enum Manufacturer {
    Hikvision = "Hikvision",
    Dahua = "Dahua"
}

File: Data/Camera.cs
using OpenAlprWebhookProcessor.Cameras.Configuration;
using System;

namespace OpenAlprWebhookProcessor.Data
{
    public class Camera
    {
        public Guid Id { get; set; }

        public double Latitude { get; set; }

        public double Longitude { get; set; }

        public CameraManufacturer Manufacturer { get; set; }

        public string OpenAlprName { get; set; }

        public long OpenAlprCameraId { get; set; }

        public string CameraPassword { get; set; }

        public string CameraUsername { get; set; }

        public string UpdateOverlayTextUrl { get; set; }
    }
}
